### DISCUSSION TITLE
Where to track usage of Github Storage? (GitHub Actions and Packages)

### DISCUSSION THREAD

**Discussion by tesar-tech** [community/community#26797] [Actions]
<p>I am receiving e-mails like: <code>You've used 75% of included services for the &lt;my account&gt; account</code>.</p>
<p>But is there any place I can see the percentage (for example after deleting some artifacts)??</p>

**Comment by laughedelic** [comment, marked as answer]
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/billing/managing-billing-for-github-actions/viewing-your-github-actions-usage">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/billing/managing-billing-for-github-actions/viewing-your-github-actions-usage" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/766656/181099945-89c46028-c3f6-4c4b-a273-f4ca5da5e93d.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/billing/managing-billing-for-github-actions/viewing-your-github-actions-usage" target="_blank" rel="noopener nofollow ugc">Viewing your GitHub Actions usage - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
